stratosphere/LogManager/LogManager.json

```diff
@@ -0,0 +1,103 @@
+{
+    "name": "LogManager",
+    "title_id": "0x0100000000000015",
+    "title_id_range_min": "0x0100000000000015",
+    "title_id_range_max": "0x0100000000000015",
+    "main_thread_stack_size": "0x00004000",
+    "main_thread_priority": 10,
```

Comment:
This is incorrect. LogManager.MainThread has a system thread priority of 10; this corresponds to an actual hos priority of 38.